app/lib/router/router.dart

```diff
@@ -359,6 +361,54 @@ List<RouteBase> makeRoutes(Ref ref) {
         );
       },
     ),
+    GoRoute(
+      parentNavigatorKey: rootNavKey,
+      name: Routes.linkChat.name,
+      path: Routes.linkChat.route,
+      pageBuilder: (context, state) {
+        return SideSheetPage(
+          key: state.pageKey,
+          transitionsBuilder: (context, animation, secondaryAnimation, child) {
+            return SlideTransition(
+              position: Tween(
+                begin: const Offset(1, 0),
+                end: const Offset(0, 0),
+              ).animate(
+                animation,
+              ),
+              child: child,
+            );
+          },
+          child: LinkChatPage(
+            parentSpaceId: state.uri.queryParameters['spaceId'],
```

Comment:
We have it in the path...

```suggestion
            parentSpaceId: state.pathParameters['spaceId'],
```